autoload/lightline/bufferline.vim

```diff
@@ -16,6 +16,7 @@ let s:number_separator    = get(g:, 'lightline#bufferline#number_separator', ' '
 let s:ordinal_separator   = get(g:, 'lightline#bufferline#ordinal_separator', '')
 let s:unnamed             = get(g:, 'lightline#bufferline#unnamed', '*')
 let s:reverse_buffers     = get(g:, 'lightline#bufferline#reverse_buffers', 0)
+let s:reverse_buffer_icons     = get(g:, 'lightline#bufferline#reverse_buffer_icon', 0)
```

Comment:
There is an **s** missing for the global variable name (icon => icon**s**). I also adjusted the spacing.
```suggestion
let s:reverse_buffer_icons = get(g:, 'lightline#bufferline#reverse_buffer_icons', 0)
```